features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimelineItemEventRow.kt

```diff
@@ -344,13 +346,13 @@ private fun MessageSenderInformation(
         }
         // Content
         Row {
-            Avatar(senderAvatar)
+            Avatar(senderAvatar, avatarColors = avatarColors)
             Spacer(modifier = Modifier.width(4.dp))
             Text(
                 text = sender,
                 maxLines = 1,
                 overflow = TextOverflow.Ellipsis,
-                color = MaterialTheme.colorScheme.primary,
+                color = avatarColors.foreground,
                 style = ElementTheme.typography.fontBodyMdMedium,
```

Comment:
Is this font right? I think it should be bold instead looking at the designs.

Reply:
This PR just deal with the colors for now... I did not have access to Figma [file](https://www.figma.com/file/rTaQE2nIUSLav4Tg3nozq7/Compound-Web-Components?type=design&node-id=522-742&mode=design&t=dgnm5jgDarXUZnEC-0), but it's OK now, I will iterate.

Reply:
We don't have the `Inter` font (I guess that's web-only), but we can set Roboto to Bold. Since the font size should probably be calculated given the size of the avatar component, that could be enough. It would be great to have some actual desgins for Android though...